libs/membership/website/src/lib/subscribe/subscribe.tsx

```diff
@@ -378,6 +390,14 @@ export const Subscribe = <T extends Exclude<BuilderUserFormFields, 'flair'>>({
 
   return (
     <SubscribeWrapper className={className} onSubmit={onSubmit} noValidate>
+      {!hasUser && returningUserId && (
+        <SubscribeSection>
+          <H5 component="h2">
+            {`Hallo ${defaults?.firstName ?? ''} ${defaults?.name ?? ''}`.trim()}, willkommen
```

Comment:
Don't think we need to trim, no? since it's note code multiple spaces should be turned into 1 or am I wrong?